src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTokenExtensions.vb

```diff
@@ -237,9 +238,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
                       token)
         End Function
 
+        Private ReadOnly Property _Facts As VisualBasicSyntaxFactsServiceFactory.VisualBasicSyntaxFactsService = VisualBasicSyntaxFactsServiceFactory.GetService
```

Comment:
I'd rather not have the local here.

In any case, convention is that private fields have _ and then a lower case first word.